### PR TITLE
less renaming

### PR DESCRIPTION
**Pull request by DavidCaughell**
not sure why core_platform was written that way

### CODE REVIEW

**Comment by inclement** [comment]
I'm not sure either, but the fix is certainly welcome. Thanks!

**Comment by dessant** [comment]
It's most likely to avoid confusion with the stdlib [platform](https://docs.python.org/3.6/library/platform.html). Maybe at a certain point both were used in the module.

File: kivy/app.py [hunk 1]
```diff
@@ -323,15 +323,12 @@ def on_resume(self):
 from kivy.event import EventDispatcher
 from kivy.lang import Builder
 from kivy.resources import resource_find
-from kivy.utils import platform as core_platform
+from kivy.utils import platform as platform
```
Comment (rnixx) [inline kivy/app.py]: Won't be ``from kivy.utils import platform`` enough?

Comment (DavidCaughell) [reply kivy/app.py]: Thanks.  :)

**Pull request merged**